keylime/cloud_verifier_tornado.py

```diff
@@ -570,14 +556,29 @@ def post(self):
 
                     # Write IMA policy to database if needed
                     if not ima_policy_name and not ima_policy:
-                        logger.info("Allowlist data not provided with request! Using default empty allowlist.")
-                        ima_policy = json.dumps(ima.process_ima_policy(ima.EMPTY_ALLOWLIST, []))
+                        logger.info("IMA policy data not provided with request! Using default empty IMA policy.")
+                        ima_policy = json.dumps(ima.EMPTY_IMA_POLICY)
 
                     if ima_policy:
-                        err_msg = cloud_verifier_common.validate_ima_policy_data(ima_policy)
-                        if err_msg:
-                            web_util.echo_json_response(self, 400, err_msg)
-                            logger.warning(err_msg)
+                        ima_policy_sig = json_body.get("ima_policy_sig")
+                        ima_policy_key = json_body.get("ima_policy_key")
```

Comment:
We need to document those options in the API docs somewhere. Once we move to DSSE the plan was to remove them again (at least the signature option), right?

Reply:
@THS-on For now, these options are the same as before, at least from the tenant API's perspective. Do we have documentation for the verifier API?

Reply:
> Do we have documentation for the verifier API?

Yes, here: https://github.com/keylime/keylime/blob/master/docs/rest_apis.rst

Reply:
@THS-on My latest commit adds documentation on the new payload fields. It also adds missing documentation on the Allowlist API from earlier, for completeness!